src/level2/reader.py

```diff
@@ -105,6 +105,9 @@ def read_profile(
         item.attrs["description"] = profile.description
         item.attrs["name"] = profile_name
 
+        if source.groupings is not None:
+            item.attrs["groupings"] = source.groupings
```

Comment:
As above, instead of `groupings` call it `attributes` and then change this to `item.attrs.update(source.attributes)` to make it more generic.